content/_changelogs/12.0.0.md

```diff
@@ -1,20 +1,167 @@
-  `on`.
-- Cypress now throws an error if commands are invoked from inside a `.should()`
-  callback. This previously resulted in unusual and undefined behavior; it is
-  now explicitly an error.
-- `Cookies.defaults` and `Cookies.preserveOnce` have been removed. Please update
-  to use [`cy.session()`](/api/commands/session) to preserve session details
+- Cypress dropped support for Node 12. Use Node 14, Node 16 or Node 18+.
+  Addressed in [#24885](https://github.com/cypress-io/cypress/pull/24885).
+- The `Cookies.defaults` and `Cookies.preserveOnce` APIs have been removed. Use
+  the new [`cy.session()`](/api/commands/session) command to preserve cookies
   between tests. Addresses
   [#21472](https://github.com/cypress-io/cypress/issues/21472).
-- The` cy.server()` and` cy.route()` commands and the `Cypress.Server.defaults`
-  API has been removed. Use [`cy.intercept()`(/api/commands/intercept) instead.
-  Addressed in [#24411](https://github.com/cypress-io/cypress/pull/24411).
+- The `cy.server()` and `cy.route()` commands have been removed. Additionally,
+  the corresponding `Cypress.Server.defaults` API has also been removed. Use the
+  [`cy.intercept()`(/api/commands/intercept) command to stub network responses
+  and requests. Addresses
+  [#22126](https://github.com/cypress-io/cypress/issues/22126).
+- The Cookie commands now uses the `hostname` as the domain by default instead
+  of the `superdomain`. This change aligns Cypress' cookie rules with the
+  browser cookie rules. This may affect what cookies are returned by
+  [`cy.getCookie()`](/api/commands/getcookie), what cookies are set with
+  [`cy.setCookie()`](/api/commands/setcookie), and the cookies cleared with
+  [`cy.clearCookies()`](/api/commands/clearcookies). Addresses
+  [#363](https://github.com/cypress-io/cypress/issues/363),
+  [#5723](https://github.com/cypress-io/cypress/issues/5723) and
+  [#24526](https://github.com/cypress-io/cypress/issues/24526)
+- Cypress now throws an error if any Cypress commands are invoked from inside a
+  [`.should()`](/api/commands/should) callback. This previously resulted in
+  unusual and undefined behavior. Addresses
+  [#5963](https://github.com/cypress-io/cypress/issues/5963),
+  [#14656](https://github.com/cypress-io/cypress/issues/14656) and
+  [#22587](https://github.com/cypress-io/cypress/issues/22587).
+
+**Features:**
+
+- Added a new configuration option called `testIsolation`, which defaults to
+  `true`. This option determines whether or not
+  [test isolation](/guides/core-concepts/writing-and-organizing-tests#Test-Isolation)
+  is enabled to ensure a clean browser context between tests. Addressed in
+  [#22230](https://github.com/cypress-io/cypress/pull/22230).
+- The previously experimental Cypress command, called
+  [`cy.origin()`](/api/commands/origin), is now generally available in
+  end-to-end testing. This command allows testing multiple domains of different
+  origins in a single test. Addresses
```

Comment:
```suggestion
  end-to-end testing. This command allows testing multiple origins in a single test. Addresses
```

Reply:
@mschile is this technically incorrect? If so, we need to update the splashscreen & will want to double-check Ely's blog post writeup for the release: https://github.com/cypress-io/cypress/pull/24925/files#diff-e1690da5e8832a5783a622b238019fa8a00d03899e74bc558b08557ded5c4593R43

Reply:
I'm trying to look at it more from a user perspective. Yes, we are adding support for 'multiple domains of different origins', but honestly, that feels difficult to read and understand. And technically, that's not even correct since `www.cypress.io` and `docs.cypress.io` are multiple domains of different origins.